src/main/java/com/github/onsdigital/babbage/api/endpoint/content/MaxAge.java

```diff
@@ -54,6 +62,40 @@ protected boolean verifyKey(String key) {
     }
 
     protected int getMaxAge(HttpServletRequest request) throws Exception {
+        if (appConfig().babbage().isLegacyCacheAPIEnabled()) {
+            CloseableHttpClient httpClient = HttpClients.createDefault();
```

Comment:
This should be moved down next to the only place where it's actually used (above `httpClient.execute`).

You should also enclose it in a try-with-resources statement.